README.markdown

```diff
@@ -12,15 +12,15 @@ Installation:
 Usage:
 
 	upstream backend {                                                                              
-		jdomain www.baidu.com; #port=80                                                             
-		#jdomain www.baidu.com port=8080; #port=8080
+		hostname www.brightcove.com; #port=80                                                             
+		#hostname www.brightcove.com port=8080; #port=8080
 	}
 	
 	resolver 8.8.8.8; #Your Local DNS Server
                                                                                                     
-Jdomain: 
+<directive> `hostname`: 
```

Comment:
I meant `<directive>` as a placeholder :) but this works. I'd probably want to reorganize this file later anyway and probably switch it to ReST

Reply:
Oh haha I figured that's what you meant but wasn't sure. We'll reorg this file later at somepoint so I'll just leave it for now.